authorization-dotnet/EncryptingAndDecryptingStringInCSharp/EncryptingAndDecryptingStringInCSharp/StringEncryptionService.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Security.Cryptography;
+using System.Text;
+
+namespace EncryptingAndDecryptingStringInCSharp
+{
+    public class StringEncryptionService
+    {
+        private byte[] IV =
+            {
+                0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
+                0x09, 0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16
+            };
+
+        private byte[] DeriveKeyFromPassword(string password) =>
+            Rfc2898DeriveBytes.Pbkdf2(Encoding.Unicode.GetBytes(password), Array.Empty<byte>(), 1000, HashAlgorithmName.SHA384, 16);
```

Comment:
Please create "explanatory" variables for the `Pbkdf2()` method parameters. It is unclear what `Array.Empty<byte>()`, `1000`, `HashAlgorithmName.SHA384` and `16` values represent. So, can you convert your 
`DeriveKeyFromPassword()` method into a multi-line one (add curly braces) and initialize these variables with meaningful names, which will then be passed into the `Pbkdf2()` method? The code will look a lot cleaner, thanks!

Reply:
fixed